packages/core/src/baseclient.ts

```diff
@@ -313,9 +312,6 @@ export abstract class BaseClient<B extends Backend, O extends Options> implement
       ...(event.user && {
         user: normalize(event.user, depth),
       }),
-      ...(event.contexts && {
-        contexts: normalize(event.contexts, depth),
-      }),
```

Comment:
Note that this is just one of the solutions (the simplest to implement) to https://github.com/getsentry/sentry-javascript/issues/2646.

Let's discuss and decide if we want this or a different solution.

I would like to avoid sending the `lcp` as a string with JSON data, because that's harder to work with in the backend shall we start processing it.

Reply:
Can't we change it to iterate over every key in contexts and normalize in the object in there?
This change not directly related to LCP, this is a general problem we should fix.

I would also prefer to have this change in a separate PR to be able to easily revert this + see the clear motivation linking to the well-described issue.

So, revert changes in the file and we are good for this PR.

Reply:
Did you see my comment above?!

Reply:
Removed from this PR. We'll solve it elsewhere.